### PR TITLE
doc/rados/ops: edit health-checks.rst (2 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit docs/rados/operations/health-checks.rst (2 of x). PR#50674, the PR that immediately precedes this PR in the series of PRs that line-edit health-checks.rst, wrongly identified this series as having five sections. This has been rectified by using the "2 of x" formulation.

Follows https://github.com/ceph/ceph/pull/50674

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
Follows https://github.com/ceph/ceph/pull/50674.

File: doc/rados/operations/health-checks.rst [hunk 1]
```diff
@@ -285,28 +285,29 @@ OSDs
 OSD_DOWN
 ________
 
-One or more OSDs are marked down.  The ceph-osd daemon may have been stopped,
-or peer OSDs may be unable to reach the OSD over the network.  Common causes
-include a stopped or crashed daemon, a down host, or a network outage.
+One or more OSDs are marked "down". The ceph-osd daemon might have been
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:288]: backticks

File: doc/rados/operations/health-checks.rst [hunk 2]
```diff
@@ -285,28 +285,29 @@ OSDs
...
+stopped, or peer OSDs might be unable to reach the OSD over the network.
+Common causes include a stopped or crashed daemon, a "down" host, or a network
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:290]: In this case suggest removing the double quotes.

File: doc/rados/operations/health-checks.rst [hunk 3]
```diff
@@ -285,28 +285,29 @@ OSDs
...
 
 OSD_<crush type>_DOWN
 _____________________
 
-(e.g. OSD_HOST_DOWN, OSD_ROOT_DOWN)
+(for example, OSD_HOST_DOWN, OSD_ROOT_DOWN)
 
-All the OSDs within a particular CRUSH subtree are marked down, for example
-all OSDs on a host.
+All of the OSDs within a particular CRUSH subtree are marked "down" (for
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:302]: backticks

File: doc/rados/operations/health-checks.rst [hunk 4]
```diff
@@ -315,12 +316,12 @@ The OSD can be removed from the CRUSH hierarchy with:
 OSD_OUT_OF_ORDER_FULL
 _____________________
 
-The utilization thresholds for `nearfull`, `backfillfull`, `full`,
-and/or `failsafe_full` are not ascending.  In particular, we expect
-`nearfull < backfillfull`, `backfillfull < full`, and `full <
+The utilization thresholds for `nearfull`, `backfillfull`, `full`, and/or
+`failsafe_full` are not ascending. In particular, the following pattern is
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:320]: Joe Cepher might not grok what "ascending" means there.  I want to sub "monotonically increasing" but I won't swear under oath that that is ideally accessible either.  But I think the following sentence explains it well enough to use the latter.

File: doc/rados/operations/health-checks.rst [hunk 5]
```diff
@@ -332,40 +333,41 @@ The thresholds can be adjusted with:
...
 
 OSD_BACKFILLFULL
 ________________
 
-One or more OSDs has exceeded the `backfillfull` threshold or *would* exceed
-when the currently mapped backfills finish, which will prevent data from being
-allowed to rebalance to this device.  This is an early warning that rebalancing
-may not be able to complete and that the cluster is approaching full.
+One or more OSDs have exceeded the `backfillfull` threshold or *would* exceed
+it if the currently mapped backfills were to finish, which will prevent data
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:365]: I want to hyphenate currently-mapped, but don't recall your stance

File: doc/rados/operations/health-checks.rst [hunk 6]
```diff
@@ -332,40 +333,41 @@ The thresholds can be adjusted with:
...
+from rebalancing to this device. This alert is an early warning that
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:366]: s/device/OSD/

File: doc/rados/operations/health-checks.rst [hunk 7]
```diff
@@ -386,23 +388,24 @@ Utilization by pool can be checked with:
...
-* *pauserd*, *pausewr* - paused reads or writes
+* *pauserd*, *pausewr* - there are paused reads or writes
 * *noup* - OSDs are not allowed to start
-* *nodown* - OSD failure reports are being ignored, such that the
-  monitors will not mark OSDs `down`
-* *noin* - OSDs that were previously marked `out` will not be marked
-  back `in` when they start
-* *noout* - down OSDs will not automatically be marked out after the
+* *nodown* - OSD failure reports are being ignored, and that means that the
+  monitors will not mark OSDs "down"
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:397]: backticks, also line 400

**Review by anthonyeleven — APPROVED** [review]
suggestions

File: doc/rados/operations/health-checks.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/health-checks.rst:288]: I think this is a mention, not a use. This, however, is part of a use-mention confusion that is general throughout the documentation, and I will bear it in mind.

File: doc/rados/operations/health-checks.rst (hunk 5 above)
Comment (zdover23) [reply doc/rados/operations/health-checks.rst:365]: We'll include the hyphen.

File: doc/rados/operations/health-checks.rst (hunk 6 above)
Comment (zdover23) [reply doc/rados/operations/health-checks.rst:366]: Good one.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50911 - Reef backport
https://github.com/ceph/ceph/pull/50912 - Quincy backport
https://github.com/ceph/ceph/pull/50913 - Pacific backport

**Pull request merged**